src/modules/add.js

```diff
@@ -14,10 +14,20 @@ const add = (e) => {
     section.insertAdjacentElement('afterend', message);
     setTimeout(() => { message.remove(); }, 2000);
   } else {
-    const score = new Scores(name, scoreV);
+    // const score = new Scores(name, scoreV);
 
-    Interface.addScoreToList(score);
-    Store.addScore(score);
+    // Interface.addScoreToList(score);
+    // Store.addScore(score);
+
+    await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/T14uVhRrwlVGFtBvKPwB/scores/', {
+      method: 'POST',
+      body: JSON.stringify({ user: name, score: scoreV }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json())
+      .then((json) => console.log(json));
```

Comment:
- [ ] for best javascript practices kindly remove the  `console.log`  from your repo. it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website, kindly remove it 🙏